### DISCUSSION TITLE
Does not work

### DISCUSSION THREAD

**Discussion by MyTestPerson** [tailwindlabs/tailwindcss#8909] [General]
https://tailwindcss.com/docs/break-after#applying-conditionally

**Comment by arisac** [comment]
Calling for dev/maintainers:

I created a play page to test `hover:break-after-column`

https://play.tailwindcss.com/qSGe2slSDD

## My Result:

### System: macOS 12.4 (21F79), Apple M1

- ❌ Google Chrome: Version 101.0.4951.41 (Official Build) (arm64)
- ✅ Safari: Version 15.5 (17613.2.7.1.8)
- ❌ Firefox: 101.0.1 (64-bit), **Firefox** is not working even on the first one

BTW, the Browser compatibility seems terrible for `break-after` according to https://developer.mozilla.org/en-US/docs/Web/CSS/break-after#browser_compatibility
